torchstain/numpy/utils/rgb2lab.py

```diff
@@ -1,35 +1,40 @@
 import numpy as np
-
-# constant conversion matrices between color spaces: https://gist.github.com/bikz05/6fd21c812ef6ebac66e1
-_rgb2xyz = np.array([[0.412453, 0.357580, 0.180423],
-                     [0.212671, 0.715160, 0.072169],
-                     [0.019334, 0.119193, 0.950227]])
+from colour.models import illuminants
```

Comment:
For this project, we wish to use pure `numpy`, `torch`, or `tensorflow` without introducing any new libraries. Hence, using the [colour](https://pypi.org/project/colour/) library is not ideal.